keps/sig-node/2133-kubelet-credential-providers/README.md

```diff
@@ -376,46 +376,51 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
 
-TBD for beta.
+  Feature is enabled but exec plugin does not properly fetch and return credentials to the kubelet.
+  Impact is that kubelet cannot authenticate and pull credentials from those registries.
 
 * **What specific metrics should inform a rollback?**
 
-TBD for beta.
+  This feature does not have metrics.
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
 
-TBD for beta.
+  No, upgrade->downgrade->upgrade were not tested. Manual validation will be done prior to promoting this feature to beta in v1.21.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
 fields of API types, flags, etc.?**
 
-TBD for beta.
+  Yes, this feature was added to remove the in-tree kubelet credential providers for AWS, Azure and GCP.
 
 ### Monitoring Requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
 
-TBD for beta.
+  Operators can check for a kubelet config file passed into the `--image-credential-provider-config`.
+  The config has a field called `imageMatches` which indicates the images a plugin will be invoked for.
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine
 the health of the service?**
   - [ ] Metrics
     - Metric name:
     - [Optional] Aggregation method:
     - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  - [X] Other (treat as last resort)
+    - Details: the kubelet has several error-level logs for when exec plugins time out or return a non-zero exit code.
```

Comment:
This seems like a reasonable metric to build.

Reply:
added metrics